psst-gui/src/ui/lyrics.rs

```diff
@@ -31,8 +32,11 @@ fn user_top_tracks_widget() -> impl Widget<AppState> {
                 .expand_width()
                 .padding(Insets::uniform_xy(theme::grid(2.0), theme::grid(0.6)))
                 .link()
+                // 19360
+                .on_left_click(|ctx, _, c, _| ctx.submit_command(cmd::SKIP_TO_POSITION.with(c.data.start_time_ms.parse::<u64>().unwrap())))
+            
         })},
-        error_widget,
+        || Label::new("No lyrics for this song!"),
```

Comment:
Ah good, I was going to suggest this would be a good error catch. However, is there a case where we know when there are no lyrics vs. getting a bad request?